packages/php-table-backend-utils/tests/Unit/Column/Bigquery/BigqueryColumnTest.php

```diff
@@ -234,15 +235,6 @@ public function testCreateArrayColumn(
     ): void {
         $data = [
             'name' => 'age',
-            'mode' => 'NULLABLE',
-            'fields' => [],
-            'description' => 'string',
-            'maxLength' => 'string',
-            'precision' => 'string',
-            'scale' => 'string',
-            'roundingMode' => 'ROUNDING_MODE_UNSPECIFIED',
-            'collation' => 'string',
-            'defaultValueExpression' => '',
```

Comment:
tady to sem dropnul, sů tam definice které nepodporujeme a já potřebuju porovnat, že `SQLtoRestDatatypeConverter` vrací 1:1 to co vrací BQ z API.